controllers/dataprotection/utils.go

```diff
@@ -269,6 +270,31 @@ func buildDeleteBackupFilesJobNamespacedName(backup *dataprotectionv1alpha1.Back
 	return types.NamespacedName{Namespace: backup.Namespace, Name: jobName}
 }
 
+func getDefaultBackupRepo(ctx context.Context, cli client.Client) (*dataprotectionv1alpha1.BackupRepo, error) {
+	backupRepoList := &dataprotectionv1alpha1.BackupRepoList{}
+	err := cli.List(ctx, backupRepoList)
+	if err != nil {
+		return nil, err
+	}
+	var defaultRepo *dataprotectionv1alpha1.BackupRepo
+	for idx := range backupRepoList.Items {
+		repo := &backupRepoList.Items[idx]
+		if repo.Annotations[constant.DefaultBackupRepoAnnotationKey] == trueVal &&
+			repo.Status.Phase == dataprotectionv1alpha1.BackupRepoReady {
+			if defaultRepo == nil {
```

Comment:
nested a bit deep.
you can refact with following code:
```
if defaultRepo != nil {
   return ...
}
defaultRepo = repo
```

Reply:
done